api/app/App.js

```diff
@@ -199,4 +199,73 @@ module.exports = function() {
 
         global.apiTracker.log(appId,"App / Table / Get", req.url,sdk);
     }
+
+
+    //Export Database for :appID
+    global.app.get('/backup/:appId/:key/exportdb',function(req, res) {
```

Comment:
Few clients handle get with body, for those clients who dont we usually have a POST handler.